source/_patterns/02-molecules/body/_body.scss

```diff
@@ -40,6 +40,11 @@ $_config: map-merge-by-keys($_config_schemes, base, $_config_schemes, $scheme);
     @include at-media(desktop) {
       @include grid-col(12);
     }
+
+    .jcc-body__main-text {
```

Comment:
Is this to _only_ target the body text? What about the lead text & header?

Reply:
@robdecker I revised to use `u-measure(6)` on both the lead and body text. I think this will be better. It better matches the intent of the ask (text has a max-width) and will match with the line length of accordions.